solidity/contracts/converter/ConverterBase.sol

```diff
@@ -430,23 +423,13 @@ abstract contract ConverterBase is IConverter, TokenHandler, TokenHolder, Contra
     */
     function reserveBalance(IERC20Token _reserveToken)
         public

```

Comment:
how come this isn't an override anymore? it should be a part of the interface as far as I can tell